qtconsole/frontend_widget.py

```diff
@@ -138,9 +134,6 @@ class FrontendWidget(HistoryConsoleWidget, BaseFrontendMixin):
     lexer_class = DottedObjectName(config=True,
         help="The pygments lexer class to use."
     )
-    is_complete_timeout = Float(0.25, config=True,
-        help="Seconds to wait for is_complete replies from the kernel."
```

Comment:
I don't see any timeout in the new code; it does need a fallback in case the kernel doesn't reply, or doesn't reply quickly enough.